### PR TITLE
Sort button works

### PR DESCRIPTION
**Pull request by Tristan-01**

### CODE REVIEW

File: cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap [hunk 1]
```diff
@@ -2,13 +2,40 @@
...
+    onDragLeave={[Function]}
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onKeyUp={[Function]}
+    onMouseDown={[Function]}
+    onMouseLeave={[Function]}
+    onMouseUp={[Function]}
+    onTouchEnd={[Function]}
+    onTouchMove={[Function]}
+    onTouchStart={[Function]}
```
Comment (alexandrelafleur) [inline cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap:22]: what is that?

File: cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap [hunk 2]
```diff
@@ -2,13 +2,40 @@
...
+    tabIndex={0}
+    type="button"
+  >
+    <span
+      className="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
+    >
+      <div
+        className="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
+      >
```
Comment (alexandrelafleur) [inline cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap:31]: a div cannot be inside of a span

File: cryptus-app/components/market_overview/market_overview.tsx [hunk 3]
```diff
@@ -128,6 +129,83 @@ export default function MarketOverview(props: market_overview_props) {
...
+    } else if (filter == "fp_d") {
+      newPropCollectionMarketTemp.sort((a, b) =>
+        a.floor_price < b.floor_price ? 1 : -1
+      );
+      newPropCollectionFavoriteTemp.sort((a, b) =>
+        a.floor_price < b.floor_price ? 1 : -1
+      );
+    } else if (filter == "delta_a") {
+      newPropCollectionMarketTemp.sort((a, b) =>
+        a.floor_price_delta / a.floor_price <
```
Comment (alexandrelafleur) [inline cryptus-app/components/market_overview/market_overview.tsx]: change else if by switch case

Comment (alexandrelafleur) [reply cryptus-app/components/market_overview/market_overview.tsx]: break;

**Review by alexandrelafleur — CHANGES_REQUESTED** [review]

File: cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap (hunk 2 above)
Comment (turcottep) [reply cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap:31]: c'est les snapshots tests c'est auto généré

Comment (alexandrelafleur) [reply cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap:31]: rip

**Review by alexandrelafleur — APPROVED** [review]

**Pull request merged**